### ISSUE TITLE
Customizer - Improved options for title, header, and footer

### ISSUE THREAD

**Issue by blue928** [wpbakery/visual-composer-starter#188]
I'm working on a site with 53 Pages and over 1000 Posts.

I would like to suggest having more global, fine tuned control over hiding/showing the title, header, and footer options.

When I go to edit an individual page or post, I see the options to show/hide the title and show/hide the header, which works on a per page basis.

When I go to the Customizer, I see Global Theme Options. There, I have a global option to turn on/off the Header and Footer, but what about the Title? I don't want to say it's necessarily a bug, but it does feel like something is missing.

As a first suggested improvement, it would be nice to also have a global option for the Titles.  Otherwie, It's impossible to manage a site that has lots of posts or pages manually.

To take this a bit further, I would like to suggestion improved Customizer options. Currently the global options apply to all posts and pages. It would be nice to be able to control global options for teach post type. For example.

Theme Options: Pages
-- hide title (on/off)
-- hide header(on/off)
-- hide footer (on/off)
-- (etc)
Theme Options: Posts
-- hide title (on/off)
-- hide header(on/off)
-- hide footer (on/off)
-- (etc)
Theme Options: Custom Post Type**
-- hide title (on/off)
-- hide header(on/off)
-- hide footer (on/off)
-- (etc)

At that point, any post or page that needed more granular control could happen at the page / post level , exactly how its currently designed.

** this is taking it a bit farther than I'd originally wanted to suggested. But if it's possible to programmatically detect custom post types and create a customizer entry for each, that would allow for ultimate flexibility.

**Comment by mmihey** [comment]
Thanks for suggestions! Sounds very handy and reasonable.

**Comment by rtpHarry** [comment]
I agree this is needed, I've had to manually disable it for every page and it's going to confuse the client if they start creating pages themselves.
